src/cmd_result.hpp

```diff
@@ -80,9 +80,12 @@ class CommandResult
 	 * response code is most appropriate for the failure.
 	 *
 	 * @param sink The ResponseSink to which the response will be sent.
+	 * @param id The connection ID in the sink to which the response will
+	 *   be sent.
 	 * @param cmd The original command that created this CommandResult.
 	 */
 	void Emit(const ResponseSink &sink,
+	          size_t id,
```

Comment:
There isn't much point other than consistency, as afaik CommandResult::Emit is always called with something other than 0 (because it's always sent to one connection now), but k